src/pages/categoryOptionGroups/Edit.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { useQuery } from 'react-query'
+import { useParams } from 'react-router-dom'
+import { DefaultEditFormContents, FormBase } from '../../components'
+import { DEFAULT_FIELD_FILTERS, SECTIONS_MAP, useOnSubmitEdit } from '../../lib'
+import { useBoundResourceQueryFn } from '../../lib/query/useBoundQueryFn'
+import { PickWithFieldFilters } from '../../types/generated'
+import { Category } from '../../types/models'
+import CategoryOptionGroupFormFields from './form/CategoryOptionGroupFormFields'
+import { validate } from './form/categoryOptionGroupSchema'
+
+const fieldFilters = [
+    ...DEFAULT_FIELD_FILTERS,
+    'name',
+    'shortName',
+    'code',
+    'description',
+    'categoryOptions[id,displayName]',
+    'dataDimension',
+    'dataDimensionType',
+    'attributeValues[value,attribute[id]]',
```

Comment:
Minor, but can we use the constant `ATTRIBUTE_VALUES_FIELD_FILTERS`. eg.
```suggestion
    ...DEFAULT_FIELD_FILTERS,
    ...ATTRIBUTE_VALUES_FIELD_FILTERS,
    'name',
    'shortName',
    'code',
    'description',
    'categoryOptions[id,displayName]',
    'dataDimension',
    'dataDimensionType',
```